google/cloud/firestore_admin_v1/py.typed

```diff
@@ -1,2 +1,2 @@
 # Marker file for PEP 561.
-# The google-firestore-admin package uses inline types.
+# The google-cloud-firestore-admin package uses inline types.
```

Comment:
I am guessing this is not easily changed for multi-gen packages?

Reply:
I'm going to put in feature request to allow for custom package names. Sometimes I want to change the name for other reasons (to insert a hyphen in the middle of a reallylongapiname for example). Getting this right is important for `_GAPIC_LIBRARY_VERSION`.